browser/ui/webui/brave_wallet/android/android_wallet_page_ui.cc

```diff
@@ -72,24 +84,24 @@ AndroidWalletPageUI::AndroidWalletPageUI(content::WebUI* web_ui,
         base::make_span(kBraveWalletSendPageGenerated,
                         kBraveWalletSendPageGeneratedSize),
         IDR_BRAVE_WALLET_SEND_PAGE_HTML);
-
   } else if (url.path() == kWalletBuyPagePath) {
     webui::SetupWebUIDataSource(
         source,
         base::make_span(kBraveWalletFundWalletPageGenerated,
                         kBraveWalletFundWalletPageGeneratedSize),
         IDR_BRAVE_WALLET_FUND_WALLET_PAGE_HTML);
-
   } else if (url.path() == kWalletDepositPagePath) {
     webui::SetupWebUIDataSource(
         source,
         base::make_span(kBraveWalletDepositPageGenerated,
                         kBraveWalletDepositPageGeneratedSize),
         IDR_BRAVE_WALLET_DEPOSIT_PAGE_HTML);
+#endif  // BUILDFLAG(ENABLE_BRAVE_ANDROID_WEB_WALLET)
   } else {
     NOTREACHED() << "Failed to find page resources for:" << url.path();
   }
 
+  source->AddBoolean("isAndroid", true);
```

Comment:
I haven't really looked in to this PR, but below is needed otherwise desktop UI would be broken.
```
diff --git a/browser/ui/webui/brave_wallet/wallet_page_ui.cc b/browser/ui/webui/brave_wallet/wallet_page_ui.cc
index 18a3b2dfaa..19b8ec2ec8 100644
--- a/browser/ui/webui/brave_wallet/wallet_page_ui.cc
+++ b/browser/ui/webui/brave_wallet/wallet_page_ui.cc
@@ -82,6 +82,7 @@ WalletPageUI::WalletPageUI(content::WebUI* web_ui)
   source->AddBoolean(brave_wallet::mojom::kP3ACountTestNetworksLoadTimeKey,
                      base::CommandLine::ForCurrentProcess()->HasSwitch(
                          brave_wallet::mojom::kP3ACountTestNetworksSwitch));
+  source->AddBoolean("isAndroid", false);
   content::URLDataSource::Add(profile,
                               std::make_unique<SanitizedImageSource>(profile));
   brave_wallet::AddBlockchainTokenImageSource(profile);
```

Reply:
I'm going to fix this, thanks!

Reply:
Is it going to be broken even if we check against null in the front end (`accounts-menu.tsx`)? 

```
const isAndroid = loadTimeData.getBoolean('isAndroid') || false
```

Reply:
Thanks! Fixed by commit 👉 [Set Android boolean as false for Desktop](https://github.com/brave/brave-core/pull/20635/commits/c43accefe8991ea0d9ffd26a5002fb06fb513515).